Resources/Prototypes/Entities/Mobs/Cyborgs/base_borg_chassis.yml

```diff
@@ -124,6 +136,15 @@
     damageContainer: Inorganic
   - type: Destructible
     thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 50
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          path: /Audio/Machines/warning_beep.ogg
+          params:
+            volume: 10
```

Comment:
does this play only when taking damage? it might play on repair.

Reply:
I tested in-game by damaging then repairing a borg and damaging it again and it played both times the borg's damage went above 50, and didn't when it was healed.

Reply:
I'm pretty sure repairing a borg fully heals it at least currently, if it somehow was repaired and still had over 50 damage it possibly might play it but that's not something that currently happens